articles/user-guide/host-programs.md

```diff
@@ -315,12 +315,22 @@ A Python host program is constructed as follows:
 
     With the `qsharp` module imported, you can also import callables directly from the Q# library namespaces.
 
-3. Among any other Python code, you can now call those callables on specific target machines, and assign their returns to variables (if they return a value) for further use.
+3. Alongside regular Python code, you can now run those callables on specific target machines (see next section), and assign their return values to variables for further use:
+
+    ```python
+    random_bit = MeasureSuperposition.simulate()
+    print(random_bit)
+    ```
 
 #### Specifying target machines
 
-Calling an operation to be run on a specific target machine is done via different Python methods on the imported object.
-For example, `.simulate(<args>)`, uses the `QuantumSimulator` to run the operation, whereas `.estimate_resources(<args>)` does so on the `ResourcesEstimator`.
+Running Q# operations on a specific target machine is done by invoking Python methods directly on the imported operation object. Thus, there is no need to first create an object for the execution target such as a simulator. Call one of the following methods on an imported Q# operation to execute it:
```

Comment:
```suggestion
Running Q# operations on a specific target machine is done by invoking Python methods directly on the imported operation object. Thus, there is no need to create an object for the run target, such as a simulator. To run an imported Q# operation, invoke one of the following methods:
```

Reply:
We try to avoid 'execute' and 'execution' (for localization reasons) if at all possible, and use 'run' instead. It's not always possible, though.

Reply:
Gotcha!